components/formats-api/src/loci/formats/FormatTools.java

```diff
@@ -2076,4 +2076,9 @@ public static int getRequiredDirectories(String[] files)
 
     return (int) Math.max(maxDirCount - dirCount, 0);
   }
+
+  static boolean isOmero(String id) {
```

Comment:
Note this is now turning an API addition and would redefine the scope of the release to a minor release increment. Should we discuss it during the weekly formats meeting.

Reply:
Yeah, lets discuss at the formats meeting and make the call then.

Reply:
alternative is to have the same method as private in 2 classes.